geopyspark/tests/base_test_class.py

```diff
@@ -4,4 +4,4 @@
 
 
 class BaseTestClass(unittest.TestCase):
-    geopysc = GeoPyContext(master="local[*]", appName="test")
+    geopysc = GeoPyContext(master="local[1]", appName="test")
```

Comment:
Does it? Do you think that trend will continue as we add more tests?

Personally, I run a lot of tests locally, but getting them to run on Travis well is also important. I think it might be best to keep it at certain level for both local and Travis for now, but then check on the performance as the tests increase/become more complicated.

What is your opinion @echeipesh?